UnityProject/Assets/Scripts/Items/Devices/ButtonSignalReceiver.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Communications;
+using Managers;
+using Mirror;
+using Objects.Wallmounts;
+
+///NOTE : THIS ITEM IS CURRENTLY HERE FOR TESTING REASONS ONLY
+///PLEASE REMOVE THIS (OR POLISH IT AND ADD IT TO THE LIST OF THINGS SCIENCE CAN MAKE) ONCE WE MAKE THE MOVE TO FULLY USE THE SIGNAL MANAGER
+namespace Items
+{
+	public class ButtonSignalReceiver : SignalReceiver, ICheckedInteractable<HandApply>
+	{
+		private DoorSwitch _switch;
```

Comment:
```suggestion
		private DoorSwitch switch;
```

Reply:
plain `switch` causes compiler errors

Reply:
yeah, because it is a reserved word. Use ``doorSwitch`` instead.